### PR TITLE
production deploy

### PR DESCRIPTION
**Pull request by onyijne**

### CODE REVIEW

**Review by onyijne — COMMENTED** [review]
This domain `.careerbuddy-landing.netlify.app` will break in production.
I will suggest domain is written in env file so production value will be different from staging and also different from local.
`FRONTEND_URL=.localhost.com` for local env
 file
`FRONTEND_URL=.careerbuddy-landing.netlify.app` for staging env file
`FRONTEND_URL=.thecareerbuddy.com`  for production env file

**Review by onyijne — COMMENTED** [review]
Line 114 will only trigger a notification if check fails, but won't stop the process.
Also, the `check.length == 2` is redundant in line 87 and 96. Doing just 
```js
if (

File: src/utils/helper.js
Comment (onyijne) [inline src/utils/helper.js:1]: This domain .careerbuddy-landing.netlify.app will break in production.
I will suggest domain is written in env file so production value will be different from staging and also different from local.
FRONTEND_URL=.localhost.com for local env
file
FRONTEND_URL=.careerbuddy-landing.netlify.app for staging env file
FRONTEND_URL=.thecareerbuddy.com for production env file

File: src/views/auth/Onboard.vue
Comment (onyijne) [inline src/views/auth/Onboard.vue:1]: Line 114 will only trigger a notification if check fails, but won't stop the process.
Also, the check.length == 2 is redundant in line 87 and 96. Doing just

if (check.length != 2 && ....) 
would be fine.

**Review by careerbuddytech — CHANGES_REQUESTED** [review]
isValidLinkedIn should return true or false, and handleEvent should proceed or terminate based on the returned value in `views/auth/onboard.vue`.

`utils/helpers.js` also need to be updated to avoid the [cookie domain](https://github.com/CareerBuddyTalent/candidate/pull/37/files#r1233206389) breaking in production.

**Review by careerbuddytech — APPROVED** [review]

**Pull request merged**